cheat.php

```diff
@@ -557,12 +557,23 @@ function GetBestPlanetAndZone( &$ZonePaces, $WaitTime )
 			}
 		}
 	}
-
-	usort( $Planets, function( $a, $b )
+	
+	if( $Planet[ 'high_zones' ] == 0 && $Planet[ 'medium_zones' ] == 0 )
 	{
-		return $b[ 'sort_key' ] - $a[ 'sort_key' ];
-	} );
-
+		usort( $Planets, function( $a, $b )
+		{
+			return $a[ 'state' ][ 'capture_progress' ] < $b[ 'state' ][ 'capture_progress' ];
```

Comment:
return type of comparison function is supposed to be an int

Reply:
Why does it matter? It still works, I've tested it.

Reply:
There are no problems with it being a float

Reply:
It's not a float, you're returning a boolean true/false value, which php is parsing as a 1/0.
Standards use negative number, zero, positive number.  By returning a boolean value, you are losing the negative number.
Zero is used to denote that the values are equal.
Basically what your code is doing is: if A is greater than B, return a 0 to tell the sort method that they're equal.

Your code might be working, but it's not working for the right reasons.